pkg/cluster/k8sres.go

```diff
@@ -1052,6 +1052,11 @@ func (c *Cluster) generateCloneEnvironment(description *acidv1.CloneDescription)
 func (c *Cluster) generatePodDisruptionBudget() *policybeta1.PodDisruptionBudget {
 	minAvailable := intstr.FromInt(1)
 
+	// Is master disruption is enabled or if there is no master, set the budget to 0.
+	if c.Spec.AllowMasterDisruption || c.Spec.NumberOfInstances <= 0 {
```

Comment:
Do we create the pod disruption budget when we scale up?

Reply:
I'll test with kind (https://github.com/zalando/postgres-operator/pull/548/files#diff-6c1850e38fa56627026b37c51091de5bR261)